### PR TITLE
Clients: Pass lifetime on from add_container to add_did Fix #444

### LINKED ISSUE #444: Lifetime in add_container ignore
Motivation
----------
Lifetime is a valid parameter of add_container(), but is ignored

Modification
------------

Expected behavior
-----------------
Should be passed through to add_did()

### PR DESCRIPTION
**Pull request by ericvaandering**
Pull request title
------------------

The format of the Pull request title must be:

    <component>: <short_change_message> #<issue number>

If you add a [github-recognised keyword](https://help.github.com/articles/closing-issues-using-keywords/) then
the associated issue can be closed automatically once the pull request is merged, e.g.::

    <component>: <short_change_message> Fix #<issue number>

Valid component names are listed in the `label list <https://github.com/rucio/rucio/labels>`_.

### CODE REVIEW

**Review by mlassnig — DISMISSED** [review]

File: lib/rucio/client/didclient.py [hunk 1]
```diff
@@ -150,11 +150,11 @@ def add_container(self, scope, name, statuses=None, meta=None, rules=None, lifet
...
         :param name: The data identifier name.
         :param statuses: Dictionary with statuses, e.g.g {'monotonic':True}.
-        :meta: Meta-data associated with the data identifier is represented using key/value pairs in a dictionary.
-        :rules: Replication rules associated with the data identifier. A list of dictionaries, e.g., [{'copies': 2, 'rse_expression': 'TIERS1'}, ].
+        :param meta: Meta-data associated with the data identifier is represented using key/value pairs in a dictionary.
+        :param rules: Replication rules associated with the data identifier. A list of dictionaries, e.g., [{'copies': 2, 'rse_expression': 'TIERS1'}, ].
         :param lifetime: DID's lifetime (in seconds).
         """
-        return self.add_did(scope=scope, name=name, type='CONTAINER', statuses=statuses, meta=meta, rules=rules)
+        return self.add_did(scope=scope, name=name, type='CONTAINER', statuses=statuses, meta=meta, rules=rules, lifetime=lifetime)
```
Comment (mlassnig) [inline lib/rucio/client/didclient.py:158]: @ericvaandering You can add yourself as author in the header of this file

Comment (vingar) [reply lib/rucio/client/didclient.py]: ... and in the main AUTHORS file as well

Comment (vingar) [reply lib/rucio/client/didclient.py]: oups I've just seen that it's there already :) deleting this comment

**Comment by ericvaandering** [comment]
Done, squashed, and pushed.

**Comment by ericvaandering** [comment]
@vingar adding my name was in the CONTRIBUTING.rst file, so I did that. :-)

**Review by mlassnig — APPROVED** [review]

**Pull request merged**